test/api/unit/middlewares/errorHandler.test.js

```diff
@@ -123,7 +123,7 @@ describe('errorHandler', () => {
     expect(res.json).to.be.calledWith({
       success: false,
       error: 'BadRequest',
-      message: 'Invalid request parameters.',
+      message: 'invalid param',
```

Comment:
Update unit test to expect the actual invalid parameter instead of the generic message.